ociclient/cache/cache_suite_test.go

```diff
@@ -76,25 +77,25 @@ var _ = Describe("Cache", func() {
                 # TYPE ociclient_cache_items_total gauge
                 ociclient_cache_items_total{path="%s"} 1
```

Comment:
```suggestion
                ociclient_cache_items_total{id="%s"} 1
```
the test for the metrics are failing. can you adjust the expected metric string